.github/CHANGELOG.md

```diff
@@ -32,6 +32,8 @@
 * Corrects bug in the function `photon_number_covar` that gave incorrect results when the covariance between two modes with finite displacements was calculated. 
 [#264](https://github.com/XanaduAI/thewalrus/pull/264)
 
+* Fixes a bug in `setup.py` that would cause the build to fail when using miniforge for M1 macs.
```

Comment:
So Jack and I tested this last night on my M1, and it built fine, but when we ran the tests it could not find thewalrus.libwalrus for some reason.

Reply:
@tlubowe, I get the same bug if I do `pip install .` instead of `pip install -e .`. Try doing the latter and see if that fixes the issue